CHANGELOG.md

```diff
@@ -3,6 +3,33 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 3.0.0 - Wanne - 2025-05-02
+
+### `Added`
+
+- [#275](https://github.com/nf-core/epitopeprediction/pull/275) - Added bcftools/stats to add MultiQC plots for variant input
+
+### `Added`
+
+- [#279](https://github.com/nf-core/epitopeprediction/pull/279) - Add `SUMMARIZE_RESULTS` module and MultiQC plots. BREAKING output structure change from`<outdir>/predictions/<meta.id>/<meta.id>.tsv` to `<outdir>/predictions/<meta.id>.tsv`
+
+### `Changed`
+
```

Comment:
A lot of modules, and scripts seems to be replaced with others. I think adding a bit more detailed changelog will help users to understand new structuring. like, running parameters change or not..